apps/prairielearn/src/components/GroupWorkInfoContainer.html.ts

```diff
@@ -0,0 +1,291 @@
+import { html } from '@prairielearn/html';
+
+import type { GroupConfig } from '../lib/db-types.js';
+import type { GroupInfo } from '../lib/groups.js';
+import { idsEqual } from '../lib/id.js';
+
+import { Modal } from './Modal.html.js';
+
+export function GroupWorkInfoCountainer({
+  groupConfig,
+  groupInfo,
+  userCanAssignRoles,
+  csrfToken,
+}: {
+  groupConfig: GroupConfig;
+  groupInfo: GroupInfo;
+  userCanAssignRoles: boolean;
+  csrfToken: string;
+}) {
+  return html`
+    <div class="container-fluid">
+      <div class="row">
+        <div class="col-sm bg-light py-4 px-4 border">
+          <h6 style="display:inline"><b>Group name: </b></h6>
+          <p style="display:inline" id="group-name">${groupInfo.groupName}</p>
+          <br />
+          ${groupConfig.student_authz_join
+            ? html`
+                <h6 style="display:inline"><b>Join code: </b></h6>
+                <p style="display:inline" id="join-code">${groupInfo.joinCode}</p>
```

Comment:
Using `h6` and `b` here, as well as a `p` with inline display, bugs me a bit. Any suggestion for something a bit more semantic?

Reply:
Agree that this is not great. I think `<p><strong>Join code:</strong> ...</p>` would be fine.

Reply:
Done in 3340238d6.